docs/collections/catalog/prefect-alert.yaml

```diff
@@ -0,0 +1,8 @@
+collectionName: prefect-alert
+author: Prefect
```

Comment:
Suggest making you the author until such time as this is moved to the prefect repos. Michael did that for his prefect-kv collection.

Reply:
Just updated the author. Thank you for reviewing!